src/apps/activity_flows/domain/update.py

```diff
@@ -0,0 +1,18 @@
+import pydantic.types as types
+from pydantic import BaseModel
+
+__all__ = ["ActivityFlowUpdate", "ActivityFlowItemUpdate"]
+
+
+class ActivityFlowItemUpdate(BaseModel):
+    id: int | None = None
+    activity_guid: types.UUID4
+
+
+class ActivityFlowUpdate(BaseModel):
+    id: int | None
+    name: str
+    description: types.Dict[str, str]
+    is_single_report: bool = False
+    hide_badge: bool = False
+    items: types.List[ActivityFlowItemUpdate]
```

Comment:
The same about the typing as above. Fell free to resolve all of them after replying at least in one